packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -236,7 +238,7 @@ contract Exchange is IExchange, Initializable, Ownable, UsingRegistry {
     * @notice Allows owner to set the minimum number of reports required
     * @param newMininumReports The new update minimum number of reports required
     */
-  function setMinimumReports(uint256 newMininumReports) public onlyOwner {
+  function setMinimumReports(uint256 newMininumReports) external onlyOwner {
```

Comment:
`setUpdateFrequency` above is still `public`.

Although I'm thinking that instead we should make these `public` and use them in `initialize` (and add setters for the parameters that don't have them yet).

Reply:
I think most of why there's some inconsistencies here is because I rushed a bit when originally getting this code in, sorry! 

We want setters so that the parameters are easily controllable via governance. In the setters we should emit events, and that's why we don't want to set these values directly in `initialize` - to deduplicate all the additional logic that happens when setting a parameter (e.g. emitting an event).

I use `_` for parameters that would otherwise have the same name as a storage variable. Since fraction elements get put into a struct, but have to be provided separately as arguments, there's no clash. Although this is going to change when #337 makes it in.